<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EzPuzzles</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="GameWidget" name="gameWidget">
       <property name="geometry">
        <rect>
         <x>111</x>
         <y>120</y>
         <width>400</width>
         <height>276</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="FormFinalImage" name="widgetFinalImage"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewGame"/>
   <addaction name="actionSaveLoad"/>
   <addaction name="separator"/>
   <addaction name="actionRestart"/>
   <addaction name="actionFinalImage"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionNewGame">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/newGame</normaloff>:/icons/newGame</iconset>
   </property>
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/exit</normaloff>:/icons/exit</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFinalImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/success</normaloff>:/icons/success</iconset>
   </property>
   <property name="text">
    <string>Final image</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/restart</normaloff>:/icons/restart</iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionSaveLoad">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/saveLoad</normaloff>:/icons/saveLoad</iconset>
   </property>
   <property name="text">
    <string>Save/Load</string>
   </property>
   <property name="toolTip">
    <string>Save/Load game</string>
   </property>
  </action>
  <action name="actionFromHistory">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/history</normaloff>:/icons/history</iconset>
   </property>
   <property name="text">
    <string>Open an Image from history</string>
   </property>
   <property name="toolTip">
    <string>Open an Image from history</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/plus</normaloff>:/icons/plus</iconset>
   </property>
   <property name="text">
    <string>Open new image</string>
   </property>
   <property name="toolTip">
    <string>Open new image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/load</normaloff>:/icons/load</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FormFinalImage</class>
   <extends>QWidget</extends>
   <header>FormFinalImage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameWidget</class>
   <extends>QWidget</extends>
   <header>GameWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFinalImage</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionFinalImage</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveGame()</slot>
  <slot>loadGame()</slot>
 </slots>
</ui>
